### PR TITLE
docs: Fix lots of typos and grammar errors

### PR DESCRIPTION
**Pull request by mowies**
This PR fixes lots of typos and grammar errors in the tutorials

### CODE REVIEW

File: site/tutorials/snippets/07/install/authCLI-istio.md [hunk 1]
```diff
@@ -31,4 +31,4 @@ If you want, you can go ahead and take a look at the Keptn API by navigating to
 echo $KEPTN_ENDPOINT
 ```
 
-![api](./assets/keptn-api.png)
\ No newline at end of file
+![api](./assets/keptn-api.png)
```
Comment (jetzlstorfer) [inline site/tutorials/snippets/07/install/authCLI-istio.md:34]: the preview does not render this file, is it available?

File: site/tutorials/snippets/07/install/open-bridge.md [hunk 2]
```diff
@@ -22,4 +22,4 @@ Positive
 
 Besides the Keptn's Bridge, please consider also taking a look at the Keptn API endpoint if you are interested to interact with Keptn via the API. Keptn comes with a fully documented swagger-API that can be found under the `/api` endpoint.
 
-![api](./assets/keptn-api.png)
\ No newline at end of file
+![api](./assets/keptn-api.png)
```
Comment (jetzlstorfer) [inline site/tutorials/snippets/07/install/open-bridge.md:25]: same as previous comment

File: site/tutorials/snippets/08/install/authCLI-istio.md [hunk 3]
```diff
@@ -31,4 +31,4 @@ If you want, you can go ahead and take a look at the Keptn API by navigating to
 echo $KEPTN_ENDPOINT
 ```
 
-![api](./assets/keptn-api.png)
\ No newline at end of file
+![api](./assets/keptn-api.png)
```
Comment (jetzlstorfer) [inline site/tutorials/snippets/08/install/authCLI-istio.md:34]: file is not rendered correctly

File: site/tutorials/snippets/08/install/open-bridge.md [hunk 4]
```diff
@@ -22,4 +22,4 @@ Positive
 
 Besides the Keptn's Bridge, please consider also taking a look at the Keptn API endpoint if you are interested to interact with Keptn via the API. Keptn comes with a fully documented swagger-API that can be found under the `/api` endpoint.
 
-![api](./assets/keptn-api.png)
\ No newline at end of file
+![api](./assets/keptn-api.png)
```
Comment (jetzlstorfer) [inline site/tutorials/snippets/08/install/open-bridge.md:25]: file not rendered correctly

**Review by jetzlstorfer — CHANGES_REQUESTED** [review]
please have a look at one file/image that is not rendered correctly

**Comment by jetzlstorfer** [comment]
I realized that in the generated preview, the image is available https://deploy-preview-181--keptn-tutorials.netlify.app/tutorials/keptn-full-tour-dynatrace-08/index.html?index=..%2F..index#6 

However, the screenshot shows an old API version. @mowies could you please update the image for the 0.8.x versions? (we can keep the old image for the old versions of course)

**Comment by mowies** [comment]
@jetzlstorfer I added a new screenshot of API version 0.8.7 and replaced it in the correct files. You can check it [here](https://deploy-preview-181--keptn-tutorials.netlify.app/tutorials/keptn-full-tour-dynatrace-08/index.html?index=..%2F..index#6)

**Review by jetzlstorfer — APPROVED** [review]
lgtm

**Pull request merged**